### ISSUE TITLE
vterm_cmd: Failed to find command: (most emacs commands)

### ISSUE THREAD

**Issue by MitchMarq42** [akermu/emacs-libvterm#597]
Actual behavior:
- execute `vterm_cmd find-file-other-window .`
- get error in echo area: `Failed to find command: find-file-other-window`

Expected behavior:
- execute `vterm_cmd find-file-other-window .`
- Dired buffer of current directory opens in the other window (as with C-x 4 f or M-x find-file-other-window)

this also happens with:
- `emacs-version`
- `update-pwd`
- `ibuffer`
- Most other commands

but not with:
- `find-file`

Vterm installed with straight.el
use-package configuration:
```elisp
(use-package vterm
  :custom
  (vterm-always-compile-module t)
  (vterm-module-cmake-args "-DUSE_SYSTEM_LIBVTERM=no")
  (vterm-clear-scrollback-when-clearing t)
  :init
  (add-to-list 'vterm-eval-cmds
	       '("update-pwd"
		 (lambda (path)
		   (setq default-directory path))))
  :hook
  (vterm-exit-functions save-buffers-kill-terminal)
  (vterm-mode . mitch/terminal-setup))
```
(`mitch/terminal-setup` is a wrapper function that turns off line numbers.)

Shell:`zsh 5.9 (x86_64-pc-linux-gnu)`

Shell-side configuration:

```sh
  if [[ "$INSIDE_EMACS" = 'vterm' ]] \
         && [[ -n ${EMACS_VTERM_PATH} ]] \
         && [[ -f ${EMACS_VTERM_PATH}/etc/emacs-vterm-bash.sh ]]; then
      source ${EMACS_VTERM_PATH}/etc/emacs-vterm-bash.sh &&
          alias emacs="vterm_cmd find-file"
  fi
  vterm_set_directory() {
      vterm_cmd update-pwd "$PWD/"
  }
  autoload -U add-zsh-hook
  add-zsh-hook -Uz chpwd (){ vterm_set_directory }
```
(this is the snippet from the README with an added alias for `emacs`.)

Emacs version:
`GNU Emacs 29.0.50 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.34, cairo version 1.17.6) of 2022-06-03`

OS: `Arch Linux x86_64`
WM/DE: Sway on wlroots/wayland

**Comment by Sbozzolo** [comment]
Due to security reasons, `vterm` only executes only commands that are explicitly whitelisted in the variable `vterm-eval-cmds`. So, if you append `find-file-other-window` to the variable, the command should go through. 

I will make sure that an informative error message is produced when the command is not in the list.

**Comment by Sbozzolo** [comment]
Fixed in [3155a47](https://github.com/akermu/emacs-libvterm/commit/3155a477b43c1567d754768f5be79296440ebaf8).

Please, reopen if you have problems.

**Comment by MitchMarq42** [comment]
Works. Thank you.
